pyproject.toml

```diff
@@ -112,15 +110,37 @@ dev = [
     "black>=20.8b1",
     "docutils",
 ]
+# Algorithms
+algorithms = [  # tested algorithms
+    "scanpy[leiden]",
+    "scanpy[magic]",
+    "scanpy[skmisc]",
+    "scanpy[harmony]",
+    "scanpy[scanorama]",
+    "scanpy[scrublet]",
+]
+algorithms-full = [
+    "scanpy[algorithms]",
+    "scanpy[louvain]",
+    "scanpy[bbknn]",
+]
```

Comment:
```suggestion
```

I think these are a bit much. I'd be up for `test` and `test-min`/ `test-full` or something along those lines.

It's just a lot of options that we're not going to maintain. Also I would like to remove a number of these.

Reply:
I think it’s useful to enable people to install `scanpy[feature]` to get a feature.

And once we add the test extras, people will start to add `scanpy[test]` to get a “default feature set” which I think is ugly.

So I’ll do it basically as you want but a bit different.